physics/module_MYNNPBL_wrapper.F90

```diff
@@ -361,20 +364,20 @@ SUBROUTINE mynnedmf_wrapper_run(        &
       endif
 
       !initialize arrays for test
-      qgrs_smoke_conc = 1.0
-      qgrs_dust_conc  = 1.0
-      FRP         = 0.
+      !qgrs_smoke_conc = 1.0
+      !qgrs_dust_conc  = 1.0
+      !FRP         = 0.
       EMIS_ANT_NO = 0.
       vdep = 0. ! hli for chem dry deposition, 0 temporarily
-      if (mix_chem) then
-         allocate ( chem3d(im,levs,nchem) )
-         do k=1,levs
-         do i=1,im
-            chem3d(i,k,1)=qgrs_smoke_conc(i,k)
-            chem3d(i,k,2)=qgrs_dust_conc (i,k)
-         enddo
-         enddo
-      endif
+      !if (mix_chem) then
+      !   allocate ( chem3d(im,levs,nchem) )
+      !   do k=1,levs
+      !   do i=1,im
+      !      chem3d(i,k,1)=qgrs_smoke_conc(i,k)
+      !      chem3d(i,k,2)=qgrs_dust_conc (i,k)
+      !   enddo
+      !   enddo
+      !endif
 
```

Comment:
You can delete all of these temporary-test lines that are now commented out. Or I can delete them later. Do you want to keep EMIS_ANT_NO = 0. and vdep = 0.?

Reply:
Following your suggestions, all these temporary-test lines were deleted in the updated PR. Yes, we keep EMIS_ANT_NO = 0. and vdep = 0.

Reply:
OK. Eventually, the specification of EMIS_ANT_NO and VDEP should be moved upstream too, but it won't hurt to do that later.